public/app/core/reducers/navBarTree.ts

```diff
@@ -2,7 +2,7 @@ import { createSlice, PayloadAction } from '@reduxjs/toolkit';
 import { NavModelItem } from '@grafana/data';
 import config from 'app/core/config';
 
-const defaultPins = ['home', 'dashboards', 'explore', 'alerting'].join(',');
+const defaultPins = ['home', 'dashboards', 'explore', 'alerting', 'cfg', 'admin', 'profile', 'help'].join(',');
```

Comment:
Does this mean that Enterprise features, e.g. reporting won't be pinned by default?

Reply:
Good point! Let me think of something